### DISCUSSION TITLE
Testing Audiogridder Cubase stop playing when a plugin is opened

### DISCUSSION THREAD

**Discussion by diegoalejo15** [apohl79/audiogridder#1247] [Help]
hi, 
I´m new to AG, I´m testing it in Cubase 12, Windows 10. Have installed server and plugin in same PC.
When I add a plugin to AG plugin and try to edit it Cubase stops playing and of course I can´t edit anything without hearing.
What am I doing wrong?
Same happen with instruments and inserts

Thx

**Comment by EkimmuMaarid** [comment]
I came here because I tried AudioGridder on Windows 10 yesterday, in order to free up processor power, the program obviously does not work. 
I confirm that only two of the three plug-ins in DAW are launched, MIDI and INSTRUMENT, respectively, there is no possibility to launch a plug-in with a through passage of sound. 
Tracktion Waveform disables Sandbox plugins when trying to load the main AudioGridder plugin, although both are written in JUCE.

**Comment by PerryDStevens** [comment]
I'm having the same thing happen here with Cubase 12 and AG 1.2

**Reply by PerryDStevens** [reply]
I went into Reaper last night and AG was working as expected.   Went back into Cubase, reset all preferences, and I also turned my local network firewall back on,.  For whatever reason it started workign again.  I also went into appdata and deleted all of the files, as well as unstalling both server and plugin instances and as was good again in the world.  That's weird because the other day it was working just fine and made no changes before it stopped working.  I have yet to test today if it's working.  I'll update when I do that.
